src/Commands/ScheduleTaskCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Commands;
+
+use ReflectionMethod;
+use Tempest\Console\Console;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Container\Container;
+use Throwable;
+
+final readonly class ScheduleTaskCommand
```

Comment:
Made some name changes to this command and did some cleanup, but nothing related to the big refactor